railties/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   When encountering an Unrecognized Command Error, the developer will be given the option to
+    run of the suggested corrections instead.
```

Comment:
I found the wording a little confusing. How about:

```suggestion
*   When encountering an Unrecognized Command Error while running `rails` commands, the developer will be     
     given the option to run the suggested corrections.
```

Reply:
Thanks for the suggestion! I sometimes find writing these challenge logs harder than writing the code 😅